pkg/content/iowriter.go

```diff
@@ -0,0 +1,100 @@
+package content
+
+import (
+	"context"
+	"io"
+
+	"github.com/containerd/containerd/content"
+	"github.com/opencontainers/go-digest"
+	log "github.com/sirupsen/logrus"
+)
+
+const (
+	// Blocksize size of byte slice for each io copy
+	Blocksize = 20480
```

Comment:
should this be configurable?

Reply:
Yeah, it should. I also passed all of my logging stuff through, which is a mistake. I will clean that up too.

Reply:
OK, now it is `DefaultBlocksize` and it is an option when creating. Take a look.

Reply:
And now it is gone. It uses `io.Copy` if a blocksize is not set, an `io.CopyBuffer` if a blocksize is set. Much more golang-native-like.